src/config/read_settings.rs

```diff
@@ -20,6 +20,7 @@ struct SecureSettings {
     alpine_mirror: Option<String>,
     site_settings: BTreeMap<PathBuf, SecureSettings>,
     external_volumes: HashMap<String, PathBuf>,
+    push_image_cmd: Option<String>,
```

Comment:
Well, it believe it should be `push-image-script`. Because (a) it's not abbreviated and (b) in current implementation it's more like script rather than command: a complex expression with substitutions is expected (or a path to external script which should work as well), and not like a command like `rsync` which has well-known arguments which we can add arbitrarily.